ddtrace/_trace/tracing_plugins/base/consumer.py

```diff
@@ -0,0 +1,132 @@
+"""
+ConsumerPlugin - Base for message consumers.
+
+Extends InboundPlugin since consuming messages is an inbound operation
+(receiving data FROM a message broker/queue).
+"""
+
+from typing import TYPE_CHECKING
```

Comment:
We shouldn't use `TYPE_CHECKING` constant any more. This is used to support python2 style typing (types in comments). We should enforce python3 typing and if we do we don;t need to hid imports. 

Using this is actually a code smell because it can hide circular imports